docs/meshstack.aws.metering.md

```diff
@@ -253,3 +154,152 @@ If you would like to see the AWS costs attributed to a seller in the [chargeback
     }
 }
 ```
+
+## AWS Cost & Usage Report based approach (Deprecated)
+
+meshStack imports metering data from [AWS Cost and Usage Reports](https://aws.amazon.com/aws-cost-management/aws-cost-and-usage-reporting/)
+
+An AWS Cost and Usage Report should be created as specified in the [documentation](https://docs.aws.amazon.com/cur/latest/userguide/cur-create.html) with the following properties
+
+* Daily time granularity
```

Comment:
can we change the terminology here? Is this the interval?

Reply:
The word granularity is used in AWS docs. I think better keep it https://docs.aws.amazon.com/cur/latest/userguide/cur-create.html